systemservices/marketplace/mesg.yml

```diff
@@ -68,11 +68,7 @@ tasks:
                         type: String	
                       service: &manifestServiceData	
                         type: Object	
-                        object:	
-                          hash:	
-                            name: ""	
-                            description: ""	
-                            type: String	
+                        object:
```

Comment:
```suggestion
```

Reply:
Why do you want to remove this?

Reply:
Sorry my mistake, on review it looked like empty object